core/src/com/softwarearchitecture/math/Vector2.java

```diff
@@ -0,0 +1,95 @@
+package com.softwarearchitecture.math;
+
+public class Vector2 {
```

Comment:
Maybe change name to Vector

Reply:
I chose the name Vector2 to indicate that it has 2 dimesions, and mirrors the vector naming convention in libgdx